join_request/jameswilson.yml

```
@@ -0,0 +1,29 @@
+collective: Potoc
+member:
+  name: James Wilson
+  address: 12TNvHiRkwzYqT5UZ86cfUvBeZBjLLYUzHLa4Hix99oTrgqT
+  github: jsdw
```

please fill your matrix handle as per https://github.com/super-collective/collective-cli/pull/4#discussion_r1586380064
```suggestion
  address: 12TNvHiRkwzYqT5UZ86cfUvBeZBjLLYUzHLa4Hix99oTrgqT
  matrix:
  github: jsdw
```

---

Done!